document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -34,27 +38,32 @@ public DocumentInfo getDocumentInfo() {
             return null;
         }
 
-        return Optional.of(transaction)
+        String resourceLink =  Optional.of(transaction)
                 .map(Transaction::getResources)
                 .map(resources -> resources.get(resourceId))
                 .map(Resource::getLinks)
                 .map(links -> links.get("resource"))
-                // when abridged has been migrated to use the company-accounts api, the code for the
-                // company accounts should work for abridged, resulting in this abridged specific code
-                // qualifying for removal
-                .filter(this::isAbridged)
-                .map(accountsLinks -> new DocumentInfo())
-                .orElse(null);
+                .orElseGet(() -> {
+                    LOG.info("Unable to find resource in transaction: " + resourceId);
```

Comment:
Can you log the transaction ID here too please.

Reply:
Done